contents.json

```diff
@@ -17039,6 +17039,25 @@
       ],
       "date_added": "Sep 3 2018",
       "suggested_by": "@vpeschenkov"
+    },
+    {
+      "title": "Good Spirits",
+      "category-ids": [
+        "health"
+      ],
+      "description": "A drink tracker with weekly limits, Untappd support, and HealthKit sync.",
+      "source": "https://github.com/archagon/good-spirits",
+      "itunes": "https://itunes.apple.com/us/app/good-spirits/id1434237439?mt=8&ref=oslist",
```

Comment:
@archagon Could you replace this link by https://itunes.apple.com/app/good-spirits/id1434237439?